src/Test/DatabaseResetter.php

```diff
@@ -47,7 +47,7 @@ public static function resetDatabase(KernelInterface $kernel): void
 
         $databaseResetter->resetDatabase();
 
-        self::bootFoundry($kernel);
+        self::bootFoundry($kernel, flushGlobalState: $damaIsEnabled);
```

Comment:
we cannot directly use `self::isDAMADoctrineTestBundleEnabled()` because it checks if `StaticDriver::isKeepStaticConnections()` and at this point [it is disabled](https://github.com/zenstruck/foundry/blob/1.x/src/Test/ResetDatabase.php#L40) even if dama is used